### ISSUE TITLE
Example has a function named 'show_challenge' but it is not included in plugin 

### ISSUE THREAD

**Issue by benjamin-frazier** [dahu/vim-sm2#1]

**Comment by dahu** [comment]
Right. The "challenge" is up to the user. It could be a knowledge quiz or a skill test. The sm2 library doesn't know or care what you're measuring; it just tracks the user-provided self-assessment of how well they met the challenge.

NOTE: I didn't go on to use this sm2 plugin in anger in any of my other plugins, so I haven't stress-tested it. Feedback welcome.

**Comment by benjamin-frazier** [comment]
What sm2 plugin do you use then?

**Comment by dahu** [comment]
I don't.
I wrote vim-sm2 with the intention of using it with my https://github.com/dahu/VimGym , but I never connected them together.

In case your question was: What SRS tool do you use? Then the answer is: https://apps.ankiweb.net/
But that's not a Vim plugin, of course.

**Comment by benjamin-frazier** [comment]
Well, I really like this plug and the idea.  I used anki for a while but I think i would really like an SRS tool integrated into vim.  I'm going to use your code and the code found @ https://github.com/clavery/vim-srs to create something.  I really don't like the dependency of python in the https://github.com/clavery/vim-srs repo but I like the folding idea to hide the answers.  I also like that your repo is pure vim script with no dependencies  on python though.  Now I just need to find the time to create it.

**Comment by dahu** [comment]
Go for it.

FWIW, here's the original article that inspired VimGym: http://of-vim-and-vigor.blogspot.com/2014/04/seconds-count-not-keystrokes.html
